libs/core/src/lib/datetime/fd-date.ts

```diff
@@ -59,6 +65,18 @@ export class FdDate {
         );
     }
 
+    static isValidByParams(params: {year: number, month?: number, day?: number, hour?: number, minute?: number, second?: number}): boolean {
+        const {year, month, day, hour, minute, second} = params;
+        const yearLength = year.toString().length;
+        // 1, 2 and 4 length are correct length of year.
+        // e.g: one digital convert correctly 1 -> 2001; two digital converted correctly 23 -> 2023.
```

Comment:
native js Date converts 1 to 1901, not 2001

Reply:
![2021-07-01_12-16](https://user-images.githubusercontent.com/14183958/124099493-425f6700-da66-11eb-8be8-74ba8d2621f7.png)

Reply:
I suggest keeping it as a helper function.